components/purchase_modal/process_payment_setup.tsx

```diff
@@ -139,6 +143,45 @@ export default class ProcessPaymentSetup extends React.PureComponent<Props, Stat
         this.props.onBack();
     }
 
+    private sucessPage = () => {
+        const {error} = this.state;
+        if (this.props.isProratedPayment) {
+            const formattedButonText = (
+                <FormattedMessage
+                    defaultMessage={'Return to {team}'}
+                    id={'admin.billing.subscription.proratedPayment.buttonText'}
+                    values={{team: this.props.teamName}}
+                />
```

Comment:
@annelieseklein given that the payment modal can (and mostly is accessed) from the System Console, the button that says "Return to {team}" seems misleading because it implies that it would take you to back to the team outside of the System Console. This will actually just close the modal and leave you where you were before. Should we reconsider changing this to something else? Before it was just `Let's go`

https://www.figma.com/file/B3YyrLCLAjNfdPUNq0shQq/Cloud-Professional-Trial-Flow?node-id=1753%3A1569

Reply:
@marianunez I changed it to "Return to {team}" because I thought that the user might want to return to working within the upgraded product and go to the team instead of the system console. If we want to just have it close (same as the close icon in the corner) then "Let's go" is fine.

Reply:
Thanks Anneliese! @pablovelezvidal see above regarding this button wording. We should be able to remove any team information we are getting that is no longer needed here for the name.